pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -1079,29 +1079,22 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard, ICo
     }
 
     /**
-     * @dev Extend ownerOnly functions to include the Managed Pool control functions.
+     * @dev Enumerates all ownerOnly functions in Managed Pool.
      */
-    function _isOwnerOnlyAction(bytes32 actionId)
-        internal
-        view
-        override(
-            // The ProtocolFeeCache module creates a small diamond that requires explicitly listing the parents here
-            BasePool,
-            BasePoolAuthorization
-        )
-        returns (bool)
-    {
+    function _isOwnerOnlyAction(bytes32 actionId) internal view override returns (bool) {
         return
             (actionId == getActionId(ManagedPool.updateWeightsGradually.selector)) ||
             (actionId == getActionId(ManagedPool.updateSwapFeeGradually.selector)) ||
             (actionId == getActionId(ManagedPool.setSwapEnabled.selector)) ||
+            (actionId == getActionId(ManagedPool.setSwapFeePercentage.selector)) ||
             (actionId == getActionId(ManagedPool.addAllowedAddress.selector)) ||
             (actionId == getActionId(ManagedPool.removeAllowedAddress.selector)) ||
             (actionId == getActionId(ManagedPool.setMustAllowlistLPs.selector)) ||
             (actionId == getActionId(ManagedPool.addToken.selector)) ||
             (actionId == getActionId(ManagedPool.removeToken.selector)) ||
             (actionId == getActionId(ManagedPool.setManagementSwapFeePercentage.selector)) ||
             (actionId == getActionId(ManagedPool.setManagementAumFeePercentage.selector)) ||
+            (actionId == getActionId(BasePool.setAssetManagerPoolConfig.selector)) ||
             super._isOwnerOnlyAction(actionId);
```

Comment:
We don't actually need to call super any more now, do we?

Reply:
It was to remind us of why we're doing this.